core-backend/wasmi/src/env.rs

```diff
@@ -56,6 +55,10 @@ pub enum WasmiEnvironmentError {
     PreExecutionHandler(String),
     #[from]
     StackEnd(StackEndError),
+    #[display(fmt = "Gas counter not found or has wrong type")]
```

Comment:
Can't here appear error `Mutable globals are not supported`?

Reply:
Nope, since wasmi has been updated to 0.14.0